splinepy/io/svg.py

```diff
@@ -984,26 +1081,38 @@ def export(
         raise ValueError("fname argument must be string")
 
     # Check if user passed list
-    if (len(splines) == 1) and isinstance(splines[0], list):
-        splines = splines[0]
-
-    for spline in splines:
-        if not isinstance(spline, Spline):
-            raise ValueError("Unexpected spline type")
-        if not 1 <= spline.para_dim <= 2:
-            raise ValueError("Only lines and surfaces supported")
-        if not spline.dim == 2:
-            raise ValueError("Only 2D geometries supported")
+    if (len(objects_to_plot) == 1) and isinstance(objects_to_plot[0], list):
+        objects_to_plot = objects_to_plot[0]
+
+    for object in objects_to_plot:
+        if isinstance(object, Spline):
+            if not 1 <= object.para_dim <= 2:
+                raise ValueError("Only lines and surfaces supported")
+            if not object.dim == 2:
+                raise ValueError("Only 2D geometries supported")
+        elif isinstance(object, _GustafBase):
+            if not object.vertices.shape[1] == 2:
+                raise ValueError("Only 2D geometries supported")
+        else:
+            raise ValueError("Unexpected spline or gustaf type")
 
     # Determine bounding box of all elements
-    ctps_bounds = splines[0].control_point_bounds
-    for spline in splines[1::]:
-        ctps_bounds[0, :] = _np.minimum(
-            spline.control_point_bounds[0, :], ctps_bounds[0, :]
-        )
-        ctps_bounds[1, :] = _np.maximum(
-            spline.control_point_bounds[1, :], ctps_bounds[1, :]
-        )
+    ctps_bounds = _np.nan * _np.ones((2, 2))
+    for object in objects_to_plot:
+        if isinstance(object, Spline):
+            ctps_bounds[0, :] = _np.fmin(
+                object.control_point_bounds[0, :], ctps_bounds[0, :]
+            )
+            ctps_bounds[1, :] = _np.fmax(
+                object.control_point_bounds[1, :], ctps_bounds[1, :]
+            )
```

Comment:
```suggestion
            bounds = object.control_point_bounds
            ctps_bounds[0, :] = _np.fmin(
                bounds[0, :], ctps_bounds[0, :]
            )
            ctps_bounds[1, :] = _np.fmax(
                bounds[1, :], ctps_bounds[1, :]
            )
```
It recomputes bounds. Minor overhead, but it is an avoidable.
gus obj doesn't recompute.